vortex-array/src/data/statistics.rs

```diff
@@ -118,7 +118,7 @@ impl Statistics for ArrayData {
         let s = self
             .encoding()
             .compute_statistics(self, stat)
-            .ok()?
+            .vortex_expect("compute_statistics must not fail")
```

Comment:
Debatable but it's really annoying to debug compute_statistics errors without this.